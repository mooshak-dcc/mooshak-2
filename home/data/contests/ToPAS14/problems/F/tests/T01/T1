108 9 9
AFFFAFFFF
AVXXXTVVF
AIXVVTVVA
AVXVVVVVA
AVXVVVXVF
AVVVVVXVA
AVVVIVXVA
AVVVVVTVF
AAAAAAAAA
4
E6
N2
W3
N6
