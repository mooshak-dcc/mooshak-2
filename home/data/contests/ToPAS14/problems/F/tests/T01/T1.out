Dentro(4,1):102:ND
